nodejs/eks/cluster.ts

```diff
@@ -475,6 +475,13 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         }));
     }
 
+    let kubernetesNetworkConfig: pulumi.Output<aws.types.output.eks.ClusterKubernetesNetworkConfig> | undefined;
+    if (args.kubernetesServiceIpAddressRange) {
+        kubernetesNetworkConfig = pulumi.output(args.kubernetesServiceIpAddressRange).apply(
+            serviceIpv4Cidr => <aws.types.output.eks.ClusterKubernetesNetworkConfig>({serviceIpv4Cidr}),
+        );
+    }
```

Comment:
Technically this should be the _input_ type, and no need to cast inside the apply:


```suggestion
    let kubernetesNetworkConfig: pulumi.Output<aws.types.input.eks.ClusterKubernetesNetworkConfig> | undefined;
    if (args.kubernetesServiceIpAddressRange) {
        kubernetesNetworkConfig = pulumi.output(args.kubernetesServiceIpAddressRange).apply(
            serviceIpv4Cidr => ({ serviceIpv4Cidr })
        );
    }
```